### DISCUSSION TITLE
What about a third tab for model?

### DISCUSSION THREAD

**Discussion by AdrianoAz** [Bing-su/adetailer#182] [Ideas]
Hi! thank you for this brillant extension, if I may, I would like to share an idea, more one tab than 2, this way in a single run, we can make the face, hands and person improvements, it is, bye!

**Comment by Bing-su** [comment]
![image](https://github.com/Bing-su/adetailer/assets/37621276/2337e274-e2fd-48d7-a999-e5ba23b23c51)
😢

**Reply by AdrianoAz** [reply]
Oh man, very thank you! thanks!
